includes/utils.inc

```diff
@@ -506,6 +506,7 @@ function wf_crm_array2str($arr) {
  *   Result of API call
  */
 function wf_civicrm_api($entity, $operation, $params) {
+  \Drupal::service('civicrm')->initialize();
```

Comment:
This needs to go into `CivicrmOptions::prepare` which is being invoked for some reason during the add element process.

Reply:
Do you want me to update and test this or is this something you have already done?

Reply:
I’ll do that - and test it -I was able to reproduce this issue

Reply:
:) thanks @KarinG. Only the author and maintainers can push to PRs

Reply:
Is it already modified and pushed as a PR? would be happy to do a quick test if it is needed :)

Reply:
@jitendrapurohit - sorry - no I don’t think I got to it before holidays and still catching up. If you can make the edit - I promise it’s top of the list for me to test / merge.

Reply:
I've submitted a new one - https://github.com/colemanw/webform_civicrm/pull/242